node-lambda/lambda.js

```diff
@@ -1,40 +1,39 @@
 'use strict';
 
 const uuid = require('uuid');
-const AWS = require('aws-sdk'); // eslint-disable-line import/no-extraneous-dependencies
-AWS.config.update({region: "us-east-2"});
-const dynamoDb = new AWS.DynamoDB.DocumentClient();
+const DynamoDB = require('aws-sdk/clients/dynamodb'); // eslint-disable-line import/no-extraneous-dependencies import/no-internal-modules
+const dynamoDb = new DynamoDB.DocumentClient({ region: "us-east-2" });
 
-module.exports.create = (event, context, callback) => {
-    const data = JSON.parse(event.body);
+module.exports.create = async (event, context) => {
+  // @see https://docs.aws.amazon.com/lambda/latest/dg/nodejs-context.html
+  context.callbackWaitsForEmptyEventLoop = false;
 
-    const params = {
-        TableName: 'book',
-        Item: {
-            id: uuid.v1(),
-            name: data.name,
-            author: data.author
-        },
-    };
+  const { name, author }  = JSON.parse(event.body);
+
+  const params = {
+    TableName: 'book',
+    Item: {
+      id: uuid.v1(),
+      name: data.name,
+      author: data.author
```

Comment:
The data variable was removed in this PR.
```suggestion
      name,
      author,
```